pkg/test/framework/components/istio/operator.go

```diff
@@ -283,21 +272,39 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 		if errs := errG.Wait(); errs != nil {
 			return nil, fmt.Errorf("%d errors occurred deploying remote clusters: %v", errs.Len(), errs.ErrorOrNil())
 		}
-	}
 
-	if env.IsMultinetwork() {
-		// enable cross network traffic
-		for _, cluster := range env.KubeClusters {
-			if err := i.applyCrossNetworkGateway(cluster); err != nil {
+		if env.IsMulticluster() {
+			// For multicluster, configure direct access so each control plane can get endpoints from all
+			// API servers.
+			if err := i.configureDirectAPIServerAccess(ctx, env, cfg); err != nil {
 				return nil, err
 			}
+			for _, cluster := range env.KubeClusters {
+				if !(env.IsControlPlaneCluster(cluster) || env.IsConfigCluster(cluster)) {
```

Comment:
This condition looks wrong to me. Should it be `if env.IsRemoteCluster(cluster)`?

Reply:
We do not have `IsRemoteCluster` method defined..  Let me add a one in env